.github/workflows/doc-lint-test.yaml

```diff
@@ -231,7 +282,29 @@ jobs:
       - name: Install geoips documentation packages
         run: pip install -e .[doc]
```

Comment:
This step should be unnecessary. `doc` is already installed in the `geoips:doclinttest-stable` image. That image is intended to have everything that we need for all doc, lint, and test jobs so that the jobs themselves don't need to install anything.

Reply:
removed